src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -197,12 +193,12 @@
                     title: fullMetaData.title,
                     url: `${this.schemas.baseUrl}${fullMetaData.type}/${fullMetaData.name}`,
                     resources: {
-                        pjnz: this.findResource(fullMetaData, this.schemas.pjnz),
-                        shape: this.findResource(fullMetaData, this.schemas.shape),
-                        pop: this.findResource(fullMetaData, this.schemas.population),
-                        survey: this.findResource(fullMetaData, this.schemas.survey),
-                        program: this.findResource(fullMetaData, this.schemas.programme),
-                        anc: this.findResource(fullMetaData, this.schemas.anc)
+                        pjnz: findResource(fullMetaData, this.schemas.pjnz),
```

Comment:
I discovered that this component had a completely duplicated version of `findResource`, and it might as well use the util